test/sagemaker_tests/pytorch/inference/integration/sagemaker/test_mnist.py

```diff
@@ -118,3 +124,50 @@ def _test_mnist_distributed(
         output = predictor.predict(data)
 
         assert output.shape == (batch_size, 10)
+
+        #  Check for Cloudwatch logs
+        if verify_logs:
+            _check_for_cloudwatch_logs(endpoint_name)
+
+def _check_for_cloudwatch_logs(endpoint_name):
+    client=boto3.client('logs')
+    identifyLogStream = client.describe_log_streams(
```

Comment:
nit: Looks like we are mixing camel case and snake case most places. We generally use snake case in this codebase

Reply:
Changed all variables to snake case